source/_posts/2023-04-18-home-assistant-os-release-10.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Home Assistant OS 10: Better memory management and new board support"
+description: "Home Assistant OS 10 released with better memory management and Hardkernel ODROID-M1 board support!"
+date: 2023-04-18 00:00:00
+date_formatted: "April 18, 2023"
+author: Stefan Agner
+comments: true
+categories:
+  - Announcements
+og_image: /images/blog/2022-05-16-os8/social.png
+---
+
+<a href='https://github.com/home-assistant/operating-system/releases/tag/10.0'>
+<img title='Home Assistant OS Release 10' alt='Home Assistant OS Release 10 Logo'
+     src='/images/blog/2022-05-16-os8/social.png' style='border: 0;box-shadow: none;'>
+</a>
+<br><br>
+
+**Highlights**:
+
+- Support for Hardkernel ODROID-M1
+- Improved datadisk feature:
+  - Improved reliability
+  - Move from one datadisk to a new datadisk is now supported
```

Comment:
```suggestion
  - Move from one data disk to a new data disk is now supported
```
If I understand correctly, the feature is called *datadisk*, one word, but I think the actual hardware thing is still spelled in two words.